tests/migration/test_host_evacuate.py

```diff
@@ -0,0 +1,92 @@
+import pytest
+
+from subprocess import CalledProcessError
+from lib.common import wait_for, wait_for_not
+# The pool needs a shared SR to use `host.evacuate`
+from tests.storage.nfs.conftest import *
```

Comment:
Wildcard import is discouraged by PEP 008 https://www.python.org/dev/peps/pep-0008/#imports